modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -5,13 +5,13 @@
 // * installing_bare_metal/installing-bare-metal-network-customizations.adoc
 
 [id="installation-user-infra-machines-static-network_{context}"]
-= Static IP address examples for RHCOS kernel parameters
+= Configure advanced networking
```

Comment:
Configure -> Configuring

Reply:
I was asked to not use a gerund in this case, so I'm going to leave it as "Configure".